tile/tile.js

```diff
@@ -4,14 +4,64 @@ import _ from 'lodash';
 
 export default class %APPNAME%Tile extends Component {
 
+  // SAMPLE TILE APP
+  constructor(props) {
+    super(props)
+    this.state = {sponsor: '', date: ''}
+  }
+  // SAMPLE END
+
   render() {
+
+    // SAMPLE TILE APP
+    //
+    const data = this.props.data || {}
+    // console.log('render data', data)
+
+    this.sponsor = <span>&mdash;</span>
+    this.date = <span>&mdash;</span>
+
+    if (data.sponsor !== undefined)
+      this.sponsor = data.sponsor
+
+    if (data.date !== undefined)
+      this.date = data.date
+
+    if (data.unknown !== undefined)
+      console.log('unknown:', data.unknown)
+
+    const bC = 'mt3 ml2'
+    const pC = 'mt1 ml2'
```

Comment:
Honestly would prefer doing the `mt3 ml2` etc etc inline instead of defining them here.